tg.js

```diff
@@ -920,20 +920,20 @@ chall.name.dmg Tumira ng {0} na damage {1}
 chall.name.bzk Manira ng killstreak ng {0} beses {1}
 chall.name.cc Ikaw ay muntikang mamatay {0} na beses {1}
 chall.name.rk Makamit ang {0} na paghiganti
-event.doubleXP 2x Class XP enabled for this match
-ranked.join Joined Ranked Match. Abandoning a match will result in a temporary ban
-ranked.progress Game in Progress
-ranked.team Please join a Team
-ranked.switch Can't switch while in-game
-ranked.spectate Spectating Disabled in Ranked
-ranked.login Login to Spawn
-ranked.sync Missing Sync Data
-ranked.team.spawn Please join a Team to Spawn
-ranked.class.spawn Please select a Class to Spawn
-modes.round.win Your Team has secured this round
-modes.round.loss This round is no longer winnable
-modes.round.switch Switching Sides. Prepare for the next Round
-modes.trade.success Trade Successful
+event.doubleXP 2x Class XP ay naka-enabled para sa larong ito
+ranked.join Naka-join na sa Ranked match. Ang pag-abandona ay mag reresulta sa temporary ban. 
+ranked.progress Kasalukuyang naglalaro
+ranked.team Pumili muna ng team
+ranked.switch Hindi makapag makapagpalit habang naglalaro
+ranked.spectate Naka-disable ang panonood sa ranked
```

Comment:
928 Naka-disable and pag-spectate sa ranked

Reply:
"Pag-spectate" is quite long.